app/views/stats/index.html.slim

```diff
@@ -15,10 +15,17 @@
 
 hr
 
-= render partial: 'stats/report',
-         locals: {correspondence_type: 'Standard FOI reports',
-                  reports: @foi_reports}
+= render partial: 'stats/report', locals: {                                    \
+    correspondence_type: 'Standard FOI reports',                               \
+    reports: @reports[:foi],                                                   \
+  }                                                                            
 
-= render partial: 'stats/report',
-        locals: { correspondence_type: 'Standard SAR reports',
-                reports: @sar_reports }
+= render partial: 'stats/report', locals: {                                    \
+    correspondence_type: 'Standard SAR reports',                               \
+    reports: @reports[:sar],                                                   \
```

Comment:
We don't need backslashes if the line ends with a comma, so I think only the first one is required